samples/tfm_integration/psa_level_1/README.rst

```diff
@@ -245,43 +272,64 @@ Sample Output
       000000E0 50 45 04 65 30 2E 30 2E 30 05 58 20 BF E6 D8 6F PE.e0.0.0.X ...o
       000000F0 88 26 F4 FF 97 FB 96 C4 E6 FB C4 99 3E 46 19 FC .&..........>F..
       00000100 56 5D A2 6A DF 34 C3 29 48 9A DC 38 06 66 53 48 V].j.4.)H..8.fSH
-      00000110 41 32 35 36 02 58 20 EF FC 32 08 03 06 CA 5A 8C A256.X ..2....Z.
-      00000120 D2 93 C8 46 04 DD 45 3F CA 41 20 47 A8 F7 D4 09 ...F..E?.A G....
-      00000130 24 16 94 38 05 68 B6 A5 01 64 4E 53 50 45 04 65 $..8.h...dNSPE.e
+      00000110 41 32 35 36 02 58 20 60 C1 87 1B F6 EC CF 35 E6 A256.X `......5.
+      00000120 30 A5 8F 14 2C B8 44 F9 E7 C7 2D EE 8D 77 99 34 0...,.D...-..w.4
+      00000130 36 D3 C7 9B 47 38 C3 A5 01 64 4E 53 50 45 04 65 6...G8...dNSPE.e
       00000140 30 2E 30 2E 30 05 58 20 B3 60 CA F5 C9 8C 6B 94 0.0.0.X .`....k.
       00000150 2A 48 82 FA 9D 48 23 EF B1 66 A9 EF 6A 6E 4A A3 *H...H#..f..jnJ.
       00000160 7C 19 19 ED 1F CC C0 49 06 66 53 48 41 32 35 36 |......I.fSHA256
-      00000170 02 58 20 D5 3F 25 8F AA 5A 05 33 36 F4 D9 2C D6 .X .?%..Z.36..,.
-      00000180 11 DF 6E 1B 18 B9 03 09 37 01 9D A7 5E FC 57 32 ..n.....7...^.W2
-      00000190 B3 1A 94 3A 00 01 25 01 77 77 77 77 2E 74 72 75 ...:..%.wwww.tru
+      00000170 02 58 20 EF 44 F8 3C 37 1C 5E 74 1F 7E 0D C9 3B .X .D.<7.^t.~..;
+      00000180 23 E5 54 05 6E 64 1D 06 1C 3A 6A 8B 18 90 5C 5D #.T.nd...:j...\]
+      00000190 0C 0F D3 3A 00 01 25 01 77 77 77 77 2E 74 72 75 ...:..%.wwww.tru
       000001A0 73 74 65 64 66 69 72 6D 77 61 72 65 2E 6F 72 67 stedfirmware.org
       000001B0 3A 00 01 24 F7 71 50 53 41 5F 49 4F 54 5F 50 52 :..$.qPSA_IOT_PR
       000001C0 4F 46 49 4C 45 5F 31 3A 00 01 24 FC 72 30 36 30 OFILE_1:..$.r060
       000001D0 34 35 36 35 32 37 32 38 32 39 31 30 30 31 30 58 456527282910010X
-      000001E0 40 51 33 D9 87 96 A9 91 55 18 9E BF 14 7A E1 76 @Q3.....U....z.v
-      000001F0 F5 0F A6 3C 7B F2 3A 1B 59 24 5B 2E 67 A8 F8 AB ...<{.:.Y$[.g...
-      00000200 12 B4 2E 09 13 5B BF 35 1F ED 66 E3 36 CF DA CE .....[.5..f.6...
-      00000210 06 03 69 DF C0 DC 4D 2F 17 33 D7 5E BE 73 B9 0E ..i...M/.3.^.s..
-      00000220 08                                              .
-      [00:00:06.982,000] <inf> app: Generating 256 bytes of random data.
+      000001E0 40 53 A1 B7 9B 18 45 D4 15 4D 84 8C A6 D6 0C 10 @S....E..M......
+      000001F0 A3 88 17 E7 E7 C9 39 72 DC 32 ED A0 DB FB EA 06 ......9r.2......
+      00000200 19 55 64 59 18 95 0E 74 90 36 26 6F E7 49 56 2E .UdY...t.6&o.IV.
+      00000210 E8 0E 8D 4D C4 1C 24 1A 52 D5 69 23 DA BE B7 1F ...M..$.R.i#....
+      00000220 0B                                              .
+      [00:00:05.784,000] <inf> app: Persisting SECP256R1 key as #1
```

Comment:
Output would maybe be a little more readable if there was some way to inject some "air" between the informational strings and the hex output

Reply:
I've added a new line between informational strings and hex output. Hope that helps.